x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/async/GetAsyncStatusRequest.java

```diff
@@ -31,12 +34,18 @@ public GetAsyncStatusRequest(String id) {
     public GetAsyncStatusRequest(StreamInput in) throws IOException {
         super(in);
         this.id = in.readString();
+        if (in.getTransportVersion().onOrAfter(TransportVersions.ASYNC_SEARCH_STATUS_SUPPORTS_KEEP_ALIVE)) {
+            this.keepAlive = in.readTimeValue();
+        }
     }
 
     @Override
     public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         out.writeString(id);
+        if (out.getTransportVersion().onOrAfter(TransportVersions.ASYNC_SEARCH_STATUS_SUPPORTS_KEEP_ALIVE)) {
+            out.writeTimeValue(keepAlive);
+        }
```

Comment:
do we have mixed cluster tests that leverage this request? Also, should we add a yaml test that sets the expiration time perhaps? Additionally, we normally have unit tests for requests, to cover their serialization at least. I guess in this case we don't have those. Would you mind adding one?

Reply:
I added a yamlRestTest that uses the keep_alive param on async-search-status.

I added a `GetAsyncStatusRequestTests.java` that matches the one for `GetAsyncResultRequestTests`.

Reply:
> do we have mixed cluster tests that leverage this request?

I don't think so? I'm a little fuzzy on how those get set up and where. I don't see any in the async-search qa module. 

I see:

```
└─ $ ▶ find . -name "*MixedCluster*"
./qa/mixed-cluster/out/test/classes/org/elasticsearch/backwards/MixedClusterClientYamlTestSuiteIT.class
./qa/mixed-cluster/src/test/java/org/elasticsearch/backwards/MixedClusterClientYamlTestSuiteIT.java
./qa/mixed-cluster/src/test/java/org/elasticsearch/backwards/MixedClusterTestNodes.java
./qa/mixed-cluster/src/test/java/org/elasticsearch/backwards/MixedClusterTestNode.java
./x-pack/plugin/esql/qa/server/mixed-cluster/src/javaRestTest/java/org/elasticsearch/xpack/esql/qa/mixed/MixedClusterEsqlSpecIT.java
```

What exactly are you wanting to test?